### PR TITLE
Don't modify image's rgb color

### PR DESCRIPTION
**Pull request by claforte**
- the code (originally from Stable-DreamFusion) that loaded images changed the RGB components of the RGBA image:
  - `(1 - rgba[..., 3:])` made colors artificially bright (when alpha is between 0 and 1), resulting in halos around the silhouette
  - pre-multiplying by alpha degrades the original color

IMHO neither of these transformation are necessary, nor desirable. The colors should be preserved as-is.

- also added "bollywood_actress" images, that I generated myself using SD2.2.x.
  - her silhouette has areas with lots of semi-transparent hair
  - the outline of her hair is abnormally bright but that's SD2.2 simulating hair back-scatter... which isn't simulated in this NeRF model, so can't be accurately reconstructed in 3D

- making this changes makes very little difference on Anya and on "Bollywood actress", but I maintain a strong opinion that this is more correct

to illustrate the difference, here are results early during training (I guess around step 400):

image

- pay attention to the left-most column of each experiment. Those are the NeRF renders for each batch item. (batch size=4)
- the right side (default) shows the old code. Notice how the side of the characters' hair are artificially bright.
- the middle side removes the additive term `(1 - rgba[..., 3:])`. 
- the left side shows the result of the new code. It preserves the original colors more accurately.

- run_zero123_examples.sh has the commands I ran, although I manually updated the code in between each experiment.

### CODE REVIEW

**Comment by claforte** [comment]
@bennyguo I only tested with the zero123 system. Let me know if you have concerns about other systems.

**Comment by claforte** [comment]
BTW the results are in https://stability.wandb.io/threestudio/claforte-dont_composite ... and the loss itself didn't change noticeably (since only a tiny fraction of pixels are affected and eventually, all variants of the code converge roughly to the same result)

**Comment by claforte** [comment]
Off the top of my head, hair and other semi-transparent surfaces can arise... e.g. stained glass, a car's window, the thing wings on the baby phoenix, etc.

File: threestudio/scripts/run_zero123_examples.sh [hunk 1]
```diff
@@ -15,3 +15,68 @@ python launch.py --config configs/zero123.yaml --train --gpu 0 system.loggers.wa
     system.prompt_processor.pretrained_model_name_or_path="./load/zero123/zero123xl_last.ckpt" \
     data.random_camera.batch_size=4 data.random_camera.batch_size=4 system.optimizer.args.lr=0.05 \
     system.guidance.min_step_percent=0.1 system.guidance.max_step_percent=0.9
+
```
Comment (voletiv) [inline threestudio/scripts/run_zero123_examples.sh:18]: Possibly unnecessary for this commit, but good to have nonetheless

**Review by voletiv — APPROVED** [review]

**Comment by bennyguo** [comment]
In my opinion, the alpha compositing formulation is correct but the matting results are inaccurate. Should we just consider the area with alpha larger than some threshold as the hard mask?

**Pull request merged**